docker-compose/docker-compose-injistack/config/mimoto-bootstrap.properties

```diff
@@ -0,0 +1,31 @@
+spring.cloud.config.uri=http://inji-web:3004/
+spring.cloud.config.name=mimoto,inji
+spring.application.name=mimoto
+
+#config.server.file.storage.uri=https://raw.githubusercontent.com/mosip/mosip-config/collab1/
+config.server.file.storage.uri=http://inji-web:3004/
+
+management.endpoint.health.show-details=always
+management.endpoints.web.exposure.include=info,health,refresh
+
+# The port number on which the Mimoto service will be running
+server.port=8099
+# Base URL for all the APIs in Mimoto service
+server.servlet.context-path=/v1/mimoto
+# Sets the maximum allowed total size (in bytes) for all HTTP response headers generated by Tomcat, including the Location header for redirects
+server.tomcat.max-http-response-header-size=65536
+
+health.config.enabled=false
+
+openapi.info.title=${spring.application.name}
+openapi.info.description=${spring.application.name}
+openapi.info.version=1.0
+openapi.info.license.name=Mosip
+openapi.info.license.url=https://docs.mosip.io/platform/license
+openapi.service.servers[0].url=${mosip.api.public.url}${server.servlet.context-path}
+openapi.service.servers[0].description=${spring.application.name}
+openapi.group.name=${openapi.info.title}
+openapi.group.paths[0]=/**
+springdoc.swagger-ui.disable-swagger-default-url=true
+springdoc.swagger-ui.tagsSorter=alpha
+springdoc.swagger-ui.operationsSorter=alpha
```

Comment:
Do we really need this file, since bootstrap files are suppose to be inside the app and not feeded externally